src/ConnectionManager.ts

```diff
@@ -54,6 +56,7 @@ export class ConnectionManager {
         for (const connection of connections) {
             if (!this.connections.find(c => c.connectionId === connection.connectionId)) {
                 this.connections.push(connection);
+                this.emit('new-connection', connection);
```

Comment:
Possibly also want one for removed connections?

Reply:
Added now in a62235e925d810ce39059784ff7db7594517b22e and implemented in FeedConnection (which drove deduplicating URLs while we were at it).